policyengine_us/parameters/gov/states/nc/tax/income/credits/ctc/head_of_household.yaml

```diff
@@ -0,0 +1,26 @@
+description: North Carolina allows for the following child tax credit for head of household filers.
+brackets:
+  - threshold:
+      2015-01-01: 0
+    amount:
+      2015-01-01: 125
+      2018-01-01: 0
+  - threshold:
+      2015-01-01: 32_000
+    amount: 
+      2015-01-01: 100
+      2018-01-01: 0
+  - threshold:
+      2015-01-01: 80_000
+    amount: 
+      2015-01-01: 0
+
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: currency-USD
+  period: year
+  label: North Carolina CTC head of household filer amount
+  reference:
+  - title: North Carolina Department of Revenue - Credit for Children
+    href: https://www.ncdor.gov/taxes-forms/individual-income-tax/credit-children
```

Comment:
could you add the [legal code](https://www.ncleg.gov/EnactedLegislation/Statutes/HTML/BySection/Chapter_105/GS_105-153.10.html) that states that it was repealed

Reply:
also 2017 [intructions](https://www.ncdor.gov/documents/files/2017-d-401-individual-income-tax-instructions/open)

Reply:
all yaml files